config/application.yml

```diff
@@ -0,0 +1,13 @@
+# Add configuration values here, as shown below.
+#
+# pusher_app_id: "2954"
+# pusher_key: 7381a978f7dd7f9a1117
+# pusher_secret: abdc3b896a0ffb85d373
+# stripe_api_key: sk_test_2J0l093xOyW72XUYJHE4Dv2r
+# stripe_publishable_key: pk_test_ro9jV5SNwGb1yYlQfzG17LHK
+#
+# production:
+#   stripe_api_key: sk_live_EeHnL644i6zo4Iyq4v1KdV9H
+#   stripe_publishable_key: pk_live_9lcthxpSIHbGwmdO941O1XVU
+
+forecast_io_key: '6296ac922ea5f4b9ae850b6aab4e336a'
```

Comment:
What does this file do?

Reply:
Unfortunately left over from weather api test yesterday.  Will delete.

Reply:
Remove the hole file!